src/EventStore.Core/LogV3/LogV3Metastreams.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using EventStore.Core.LogAbstraction;
+
+namespace EventStore.Core.LogV3 {
+	public class LogV3Metastreams : IMetastreamLookup<long> {
+		public bool IsMetaStream(long streamId) =>
+			streamId % 2 == 1;
```

Comment:
@timothycoleman Should the stream ID validator also be updated to check for streamId > 1 instead of > 0?
https://github.com/EventStore/EventStore/blob/698eb69c623ce21770c096a47b1cd1d2b4733906/src/EventStore.Core/LogV3/LogV3StreamIdValidator.cs#L9

Reply:
because 1 would be the metastream of 'no stream', which doesn't make any sense, so yep! thats a great spot

Reply:
changed this to non-negative in the end, need to accept 0 in the same way that the v3 version accepts strings that dont actually correspond to streams in the database